arelle/plugin/validate/HMRC/__init__.py

```diff
@@ -29,7 +29,7 @@
     "EntityCurrentLegalOrRegisteredName", "StartDateForPeriodCoveredByReport",
     "EndDateForPeriodCoveredByReport", "BalanceSheetDate"}
 
-COMMON_MANDATORY_FRS_ITEMS = commonMandatoryItems | {
+commonMandatoryFrsItems = commonMandatoryItems | {
```

Comment:
I believe that common python formatting is actually capitalized snake case for constants such as these. So `COMMON_MANDATORY_FRS_ITEMS` should be the name. Anything that doesnt match that is what actually needs to be updated. Further below, mustHaveOneItem should be `MUST_HAVE_ONE_ITEM`.